packages/repack/react-dom@umd/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@msinternal/react-dom-umd",
+  "description": "Repack of react-dom@umd",
+  "version": "0.0.0-0",
+  "author": "Microsoft Corporation",
+  "private": true,
+  "type": "module",
+  "exports": {
+    ".": "./dist/react-dom.js"
+  },
+  "scripts": {
+    "build": "esbuild --bundle=true --chunk-names=react-dom/[name]-[hash] --entry-names=[dir]/[name] --format=esm --outdir=dist --platform=browser --splitting --sourcemap react-dom=./src/index.js",
+    "bump": "npm run bump:prod && npm run bump:dev && (npm audit fix || exit 0)",
+    "bump:dev": "PACKAGES_TO_BUMP=$(cat package.json | jq -r '(.pinDependencies // {}) as $P | (.localDependencies // {} | keys) as $L | (.devDependencies // {}) | to_entries | map(select(.key as $K | $L | contains([$K]) | not)) | map(.key + \"@\" + ($P[.key] // [\"latest\"])[0]) | join(\" \")') && [ ! -z \"$PACKAGES_TO_BUMP\" ] && npm install $PACKAGES_TO_BUMP || true",
+    "bump:prod": "PACKAGES_TO_BUMP=$(cat package.json | jq -r '(.pinDependencies // {}) as $P | (.localDependencies // {} | keys) as $L | (.dependencies // {}) | to_entries | map(select(.key as $K | $L | contains([$K]) | not)) | map(.key + \"@\" + ($P[.key] // [\"latest\"])[0]) | join(\" \")') && [ ! -z \"$PACKAGES_TO_BUMP\" ] && npm install --save-exact $PACKAGES_TO_BUMP || true",
```

Comment:
These things are getting out of hands :rofl:

Reply:
Exactly... it was pretty small script... then we have `pinDependencies` which is actually great. But the `localDependencies` made it worse. 🤣

I kind of mixed if we should go Dependabot, which will do this for us, but may hurt us as it did it wrong (esp in mono-repo.)

Should put these into a shell script under `/scripts/` or wrap it as a Node.js script. Not sure if I should do it as a GitHub Actions, I love ability to run it locally though.